pyproject.toml

```diff
@@ -9,6 +9,7 @@ python = "^3.9"
 "discord.py" = "^2.0.1"
 jishaku = "^2.5.1"
 aiohttp = "^3.8.3"
+discord-ext-paginator = {git = "https://github.com/philskillz-coder/discord-py-paginator.git"}
```

Comment:
Doing this is very risky for a major project. This could mean breaking changes and you'd not even know as you haven't specified the version.

There are 2 options to solve this:

Use the PyPi version if available.
Use specific commit hash while using git link.